test/no-inheritance.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @fileoverview Tests for no-inheritance rule
+ * @author Nicolas Feignon <nfeignon@gmail.com>
+ */
+
+'use strict';
+
+var Solium = require ('solium');
+
+var userConfig = {
+    rules: {
+        "security/no-inheritance": "error"
+    }
+};
+
+describe ('[RULE] no-inheritance', function () {
```

Comment:
Change description to `[RULE] no-inheritance: Rejections`. This is convention throughout solium.

Also add an `Acceptances` section where its just normal contract code (without inheritance) and add `errors.length.should.equal(0);` test.